source/Options.cpp

```diff
@@ -147,336 +147,164 @@ Options::Options(
       export_origins_mode_(export_origins_mode),
       propagate_across_arguments_(propagate_across_arguments) {}
 
-Options::Options(const boost::program_options::variables_map& variables) {
-  system_jar_paths_ = parse_paths_list(
-      variables["system-jar-paths"].as<std::string>(),
-      std::nullopt,
-      /* check exist */ false);
+Options::Options(const Json::Value &json){
+    system_jar_paths_ = JsonValidation::string_list(json,"system-jar-paths");
```

Comment:
I forgot we had this `parse_paths_list` function with all the extra logic.
it seems like you decided to keep these for now, except for `system_jar_paths_`.

I think in the future we should move the logic of `parse_paths_list` into the shim, but that seems out of the scope of this PR.

for now, can we keep `system-jar-paths` as a string and use `parse_paths_list` logic here?